cobc/scanner.l

```diff
@@ -323,6 +326,13 @@ AREA_A		"#AREA_A"\n
 	cobc_areacheck = 0;
 }
 
+<*>^[ ]?"#INCLUDE".*/\n {
```

Comment:
I think we shouldn't allow an optional leading space here (and also drop it for everything that is "new" like the areacheck) but already scan the trailing one.

Reply:
"new" relatively to what?

Reply:
Every internal directive newer than GC 3.1.2 - because afterwards we only generated _without_ leading space, so it is reasonable not to check those entries for an optional space.

... we could also drop it from any internal directive and make an entry in the NEWS that source previously processed may need to be adjusted to be processed by GnuCOBOL 3.3.